packages/core/test/BooleanField_test.js

```diff
@@ -324,23 +324,6 @@ describe("BooleanField", () => {
     expect(labels).eql(["No", "Yes"]);
   });
 
-  it("should support enumNames for radio widgets", () => {
-    const { node } = createFormComponent({
-      schema: {
-        type: "boolean",
-        enumNames: ["Yes", "No"],
-      },
-      formData: true,
-      uiSchema: { "ui:widget": "radio" },
-    });
-
-    const labels = [].map.call(
-      node.querySelectorAll(".field-radio-group label"),
-      (label) => label.textContent
-    );
-    expect(labels).eql(["Yes", "No"]);
-  });
-
```

Comment:
I suppose we restore this, but check for console.warn() message about deprecation